src/Data/Vec/Relation/Binary/Reasoning/Propositional.agda

```diff
@@ -0,0 +1,139 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- The basic code for equational reasoning with displayed propositional equality over vectors
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Vec.Relation.Binary.Reasoning.Propositional {a} {A : Set a} where
```

Comment:
How about renaming this `Data.Vec.Relation.Binary.Equality.Cast`?

Reply:
Done, thanks!